lib/phoenix_live_view.ex

```diff
@@ -473,13 +473,26 @@ defmodule Phoenix.LiveView do
       require Phoenix.LiveView.Renderer
       @before_compile Phoenix.LiveView.Renderer
 
+      @phoenix_live_opts opts
+      Module.register_attribute(__MODULE__, :mount, accumulate: true)
```

Comment:
To avoid conflict with user attributes, call this  `:phoenix_live_mount` or anything with the phoenix prefix. :)